### ISSUE TITLE
prevent duplicate station entries in wx station files (pre-station fetch format)

### ISSUE THREAD

**Issue by nwagenbrenner** [firelab/windninja#250]
If multiple entries occur for a given station, the simulation uses the average of the entries as the input wind. For example, if you have two rows with exactly the same info except row 1 has direction = 90 and row 2 has direction = 180, the initialization grid for direction = 135. 

I don't think this was intended behavior, right? Seems like this behavior could easily cause problems for people. We should probably prevent duplicate stations from being used in the old single time step format. This is for the old (current) format where we don't support multiple time steps.

**Comment by jforthofer** [comment]
Yeah, if I understand what you're saying correctly, I think we should throw an error or something.  If there are duplicate entries for exactly the same lat/lon?
